src/android/build.gradle

```diff
@@ -34,5 +34,6 @@ android {
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
+    compile 'com.squareup.okhttp3:okhttp:3.4.1'
```

Comment:
@petterh They have now updated to [`3.8.0`](https://github.com/facebook/react-native/commit/93a1d592d61b229d3dafe57d1590d286312cc0bf)